pr_agent/algo/ai_handlers/litellm_ai_handler.py

```diff
@@ -126,6 +135,7 @@ async def chat_completion(self, model: str, system: str, user: str, temperature:
                 "deployment_id": deployment_id,
                 "messages": messages,
                 "temperature": temperature,
+                "max_tokens": get_settings().config.max_tokens,
```

Comment:
what is this line ??

there is no such config parameter.

remove this

Reply:
you told me to change it [here](https://github.com/Codium-ai/pr-agent/pull/1047#discussion_r1687092276), as max_model_tokens wasn't right. max_model_tokens does not have have any docs. This is required for watsonx, as it only defaults to like 50 tokens for some reason.

I will make it watsonx specific